dep/recastnavigation/Recast/Source/Recast.cpp

```diff
@@ -207,7 +207,7 @@ void rcCalcBounds(const float* verts, int nv, float* bmin, float* bmax)
 	rcVcopy(bmax, verts);
 	for (int i = 1; i < nv; ++i)
 	{
-		const float* v = &verts[i*3];
+        const float* v = &verts[i * 3];
```

Comment:
Usually we don't apply TC code style to 3rd party code.

Reply:
Thanks, I was a bit uncertain myself.